storage/src/main/java/tech/pegasys/teku/storage/server/Database.java

```diff
@@ -70,7 +71,11 @@
   @MustBeClosed
   Stream<DepositsFromBlockEvent> streamDepositsFromBlocks();
 
+  Optional<ProtoArray> getProtoArrayFromDisk();
+
   void addMinGenesisTimeBlock(final MinGenesisTimeBlockEvent event);
 
   void addDepositsFromBlockEvent(final DepositsFromBlockEvent event);
+
+  void updateProtoArrayOnDisk(final ProtoArray protoArray);
```

Comment:
nit: Probably just `putProtoArray`.

Actually thinking about it, probably don't need `OnDisk` or `FromDisk` anywhere - the particular storage implementation in use might not write to disk.

Reply:
Done.